third_party/tpch-dbgen/build.c

```diff
@@ -242,9 +243,9 @@ mk_order(DSS_HUGE index, order_t * o, long upd_num, float scale)
 		r_date += s_date;
 
 
-		strcpy(o->l[lcnt].sdate, asc_date[s_date - STARTDATE]);
-		strcpy(o->l[lcnt].cdate, asc_date[c_date - STARTDATE]);
-		strcpy(o->l[lcnt].rdate, asc_date[r_date - STARTDATE]);
+		memcpy(o->l[lcnt].sdate, asc_date[s_date - STARTDATE], 10);
+		memcpy(o->l[lcnt].cdate, asc_date[c_date - STARTDATE], 10);
+		memcpy(o->l[lcnt].rdate, asc_date[r_date - STARTDATE], 10);
```

Comment:
Do we need to copy 11 bytes here as well (`\0`)?

Reply:
That was already zero-initialized, but I changed it to be safe.